packages/backend/src/scheduler/index.ts

```diff
@@ -0,0 +1,28 @@
+import schedule from 'node-schedule';
+import type { ProductRepository } from '@repositories';
+import type { Product } from '@prisma/client';
+import { AuctionNotificationsCommand } from '../commands/auction-notifications';
+
+export const clearAllJobs = () => {
+  const jobsToDelete = Object.keys(schedule.scheduledJobs);
+  for (const job of jobsToDelete) {
+    schedule.cancelJob(job);
+  }
+};
+
+export const setJob = (
+  id: string,
+  utcDate: Date | string,
+  callback: () => void,
+) => {
+  schedule.scheduleJob(id, utcDate, callback);
+};
+
+export const initAuctionJobs = (productRepository: ProductRepository) => {
```

Comment:
How would you set a job for created in runtime auction. I believe,  it make sense to have a service with Api to add a job.